src/Documentation/sidebar.json

```diff
@@ -1,4 +1,9 @@
 [
+  {
+    "slug": "learn",
+    "label": "Learn DVC",
```

Comment:
Label is too generic. Get Started, User Guide, etc - all about learning DVC.

I would prefer to make is part of the Tutorials section (and as we discussed current single Tutorial we'll be consolidated with example-pipelines and will just one of the many tutorials). With some appropriate descriptive name or multiple names that reflect different available tutorials.

Also, put additional links in examples, command references, etc - "Try this online" ...

Reply:
I renamed it from `learn` to `basics`, and from `Learn DVC` to `DVC Basics`.

But I don't think this should be a part of the Tutorials. I will explain in another comment below.

> Also, put additional links in examples, command references, etc - "Try this online" ...

I was planning to do this later, on other pull requests, since interactive examples are not finished yet (actually just the `fetch` example is available right now, but I am going to add other examples soon).